ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.7.yaml

```diff
@@ -70,8 +70,16 @@ images:
 - dockerfile_path: ci-images/Dockerfile.unit-tests
   from: assisted-service-base
   to: assisted-service-unit-test
+- dockerfile_literal: |
+    FROM base-build
+    RUN dnf install -y diffutils # required for gofmt
+  inputs:
+    assisted-service-build-base:
+      as:
+      - base-build
+  to: assisted-service-base-lint
 - dockerfile_path: ci-images/Dockerfile.lint
```

Comment:
I think you can just add diffutils to this dockerfile, instead of building another layer of base-image

Reply:
it's in the assisted-service repository, and we can even do it for master and backport it down the branches

Reply:
Makes sense, but is that alright even if this just happens for 2.7?

Reply:
yes I think so
maybe we have different base images for other branches which cause this difference? I actually didn't check

Reply:
Ok, I'll open a PR to add diffutils in the assisted-service repo. Thanks Osher!


> maybe we have different base images for other branches which cause this difference? I actually didn't check

I think so too, looks like [2.6](https://github.com/openshift/release/blob/a9d095b49f3fc9f8f14f71554c4e59b692dc6ca2/ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.6.yaml#L102) & [2.7](https://github.com/openshift/release/blob/a9d095b49f3fc9f8f14f71554c4e59b692dc6ca2/ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.7.yaml#L63) use centos stream 9
[2.8](https://github.com/openshift/release/blob/a9d095b49f3fc9f8f14f71554c4e59b692dc6ca2/ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.8.yaml#L57) and [master](https://github.com/openshift/release/blob/5d159b078ce016b5c1dbefb1b30bd74ca0711c0f/ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml#L59) uses golang-1.18